x-pack/plugins/security_solution/public/management/pages/event_filters/view/event_filters_list_page.test.tsx

```diff
@@ -175,4 +175,30 @@ describe('When on the Event Filters List Page', () => {
       });
     });
   });
+
+  describe('and the back button is present', () => {
```

Comment:
nitpick: Maybe just call this `Back button` since now it looks similar to the test description name below.

Reply:
I'm going to merge this before having more `out of date` check errors and will do this change in next pr :)